assignments/tests/factories.py

```diff
@@ -82,3 +82,31 @@ class Meta:
         model = models.SchoolClass
 
     name = factory.Sequence(lambda n: 'class_%s' % n)
+
+
+class StudentFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = models.Student
```

Comment:
I guess we didn't change this to `Answer` [after all](https://github.com/City-of-Turku/living-environment/pull/7#discussion_r112890102).. Well, I guess `Student` is fine also.